WooCommerce/Classes/ViewRelated/Dashboard/DashboardViewModel.swift

```diff
@@ -60,6 +60,8 @@ final class DashboardViewModel: ObservableObject {
 
     @Published private(set) var jetpackBannerVisibleFromAppSettings = false
 
+    @Published private(set) var isSiteEligibleToInstallJetpack = true
```

Comment:
We need a similar logic for the app settings to hide the "Install Jetpack" button.

Reply:
Great catch. Done in 59a6121ff0e211d38068c683e5749c01ec07b8f2. Thank you!